gce/deploy.sh

```diff
@@ -0,0 +1,36 @@
+# Copyright 2019 Google LLC All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -ex
+
+MY_INSTANCE_NAME='[your instance name goes here]'
+
+ZONE=us-central1-f
+
+gcloud compute instances create $MY_INSTANCE_NAME \
+    --image-family=debian-9 \
+    --image-project=debian-cloud \
+    --machine-type=g1-small \
+    --scopes userinfo-email,cloud-platform \
+    --metadata-from-file startup-script=gce/startup-script.sh \
+    --zone $ZONE \
+    --tags http-server
+
+# [START getting_started_create_firewall]
```

Comment:
Since this is the same for every language, we may not need region tags here. Same for the `teardown` script.

Reply:
Okay